R/fastMapGenerate.R

```diff
@@ -23,7 +23,7 @@
 #' @examples
 #' \dontrun{
 #' fastMapGenerate("hgnc_complete_set.txt", "symbol", "uniprot_ids",
-#'                 type = "UniProt", outName = "fastMapUniProt.rds")
+#'                 type = "UniProt", outputName = "fastMapUniProt.rds")
 #' }
```

Comment:
That looks like a bug right? If it is, we always need to add a test because apparently that part of the code was not covered. Ha - that's a QA job. Can you alert Ricardo?